spec/system/provider_interface/see_applications_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.feature 'See applications' do
+  scenario 'Provider visits application page' do
+    given_i_am_a_provider_user
+    and_my_organisation_has_new_applications
```

Comment:
Does `new` matter? I don't have a good idea of what the basic behaviour needs to be here. Is the most conservative thing to show all applications?

Reply:
It did not, fixed in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/272/commits/c120bf93428f19bd2029c07266b79cb590a32633